cut/src/cut.yml

```diff
@@ -0,0 +1,62 @@
+name: cut
+version: "0.0.0"
+author: Mats Kindahl <mats.kindahl@gmail.com>
+about: "Print selected parts of lines from each FILE to standard output.\n\nWith no FILE, or when FILE is -, read standard input."
```

Comment:
Always use the word "display" over "print", just for stylistics and consistency with other tools, there is other cases on line 41 and 53

Reply:
Done.